coq.opam

```diff
@@ -16,7 +16,8 @@ depends: [
 ]
 
 build: [
+  [ "./configure" "-prefix" prefix "-mandir" man "-libdir" "%{lib}/coq" "-coqide" "no" ]
```

Comment:
Why add this line? Isn't it redundant with the next steps?

Reply:
Yeah sorry, I should have marked this PR as in-progress.

The thing is that I wasn't expecting people to use the opam file yet, it is not ready as we need a few changes to implement the build scheme properly, anyways I will turn this PR into a fix.